.github/workflows/check.yml

```diff
@@ -1,4 +1,4 @@
-on: [push, pull_request]
```

Comment:
The importance of keeping the `pull_request` trigger is testing if the merge to the target branch will succeed. Please, try adding it back.

Reply:
Was removed since it does not exist in main

Reply:
Yes, I saw when you removed this trigger in #14, but I recommend trying to add it back to increase the test coverage.